spec/visual/templates/layout.erb

```diff
@@ -38,6 +38,10 @@
     #page-footer {
       border-top: 1px dotted
     }
+    .rouge-line-table { border-collapse: collapse }
+    .rouge-line-table tr > td { padding: 0 5px }
+    .rouge-line-table .rouge-gutter { text-align: right }
```

Comment:
What do you think about always having the line numbers be at the top of the line? I set `vertical-align: top` and was playing around with it locally.

I thought it made it easier to quickly find the start of the line but perhaps a little more difficult to 'group' the multiline lines (if that makes sense).

I suspect you're going to use this mode more than I do so am happy to defer to you.

Reply:
Honestly, I fail to see the need for adding a `vertical-align` definition since the line are set to their natural height (based on the inherited `line-height` property).

Ignoring this.

Reply:
Cheers :) If you find after a bit more use this way isn't better, happy to receive a PR switching it back!

Reply:
Actually, this is in sync with the rendering in Sublime Text. So, I'm happy with this change.
Thank you :smiley: